torch/quantization/observer.py

```diff
@@ -238,6 +238,25 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super(MinMaxObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, False,
                                                           missing_keys, unexpected_keys, error_msgs)
 
+class MovingAverageMinMaxObserver(MinMaxObserver):
+    def __init__(self, averaging_constant=0.01, **kwargs):
```

Comment:
Is averaging_constant chosen as 1/n where n is the averaging windows size? It would be good to add some comments explaining averaging_constant.

Reply:
Would it be more explicit to user if we pass the window_size as the parameter, where window_size = 1 / averaging_constant?